pages/tools.jsx

```diff
@@ -0,0 +1,151 @@
+var React = require('react');
+var Illustration = require('../components/illustration.jsx');
+var Router = require('react-router');
+var Link = Router.Link;
+var IconLinks = require('../components/icon-links.jsx');
+var IconLink = require('../components/icon-link.jsx');
+
+var ToolsIntro = React.createClass({
+  render: function() {
+    return (
+      <div>
+        <section className="intro">
+          <Illustration
+            height={204} width={204}
+            src1x="/img/pages/tools/svg/icon-teach-like-mozilla-tools.svg"
+            alt="icon toolkit"
+            className="img-circle">
+            <h1>Tools to Teach and Learn the Web</h1>
+            <h2>These tools are free and open source, and can be used in a variety of ways to teach learners how to read, write, and participate on the Web.</h2>
+          </Illustration>
+        </section>
+      </div>
+    );
+  }
+});
+
+var ToolsColumn = React.createClass({
+  propTypes: {
+    name: React.PropTypes.string.isRequired,
+    desctiption: React.PropTypes.string.isRequired,
+    link: React.PropTypes.string.isRequired,
+    src1x: React.PropTypes.string.isRequired,
+    src2x: React.PropTypes.string.isRequired,
+    activityTitle: React.PropTypes.string,
+    activityLink: React.PropTypes.string
+  },
+  render: function() {
+    var sampleActivity = null;
+    if (this.props.activityTitle && this.props.activityLink) {
+      sampleActivity =
+        <div className="sample-activity">
+          <div className="label-container"><div className="label label-success">Sample Activity</div></div>
+          <a href={this.props.activityLink}>{this.props.activityTitle}</a>
+        </div>;
+    }
+    return (
+      <div className="col-sm-4 col-md-4 col-lg-4 tools-col">
+        <Illustration
+          link={this.props.link}
+          height={150} width={200}
```

Comment:
Hmm, so this is giving the illustration an aspect ratio of 4:3, but the `webmaker@2x.jpg` is 400x336, which has an aspect ratio closer to a square, so it appears "squished" on my retina display. The non-retina logo does have the correct aspect ratio, so I suspect it looks fine on retina displays.

Hmm, it seems like there should be an easy way to simulate retina/non-retina from the browser...